apps/notepad/src/app/component/note-details/note-details.component.ts

```diff
@@ -0,0 +1,64 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup } from '@angular/forms';
+import { ActivatedRoute } from '@angular/router';
+import { NotesService } from '../../notes.service';
+import { Note } from '../notes-list/note.model';
+import { Task } from './task.model';
+
+@Component({
+  selector: 'np-note-details',
+  templateUrl: './note-details.component.html',
+  styleUrls: ['./note-details.component.scss'],
+})
+export class NoteDetailsComponent implements OnInit {
+  notesList: Note[] = [];
+  currentNoteDetail: any;
+  todos: Task[] = [];
+  createNoteDetailForm: FormGroup = new FormGroup({
+    detailTitle: new FormControl({ value: null, disabled: true }),
+    detailContent: new FormControl({ value: null, disabled: true }),
+    detailTodo: new FormControl(null),
```

Comment:
give formcontrol some type if default value is null
e.g new FormControl<string|null>(null)